tools/sst_dump_test.cc

```diff
@@ -44,6 +45,17 @@ static std::string MakeValue(int i) {
   return key.Encode().ToString();
 }
 
+static std::string MakeWideColumn(int i) {
+  std::string val = MakeValue(i);
+  std::string val1 = "attr_1_val_" + val;
+  std::string val2 = "attr_2_val_" + val;
+  WideColumns columns{{"attr_1", val1}, {"attr_2", val2}};
+  WideColumns sorted_columns(columns);
```

Comment:
This copy seems redundant; we could just call `Serialize` with the original `columns`, right?